web/src/app/stats/components/stats-table/stats-table.component.html

```
@@ -76,9 +76,12 @@
       <mat-header-cell *matHeaderCellDef mat-sort-header i18n="@@colReps"
         >Reps</mat-header-cell
       >
-      <mat-cell *matCellDef="let entry"
-        ><span>{{ entry.reps }}</span></mat-cell
-      >
+      <mat-cell *matCellDef="let entry">
+        <span>{{ entry.reps }}</span>
+        @if (entry.sets?.length > 1) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)